tests/performanceplatform/utils/test_collector_config.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from hamcrest import assert_that, equal_to
+from mock import patch
+from performanceplatform.utils.collector import get_config
+
+
+class ConfigTest(unittest.TestCase):
+    @patch('performanceplatform.client.collector.CollectorAPI.get_collector')
```

Comment:
The two tests need to be taken together. If you pass a -q (query) to the command, we check that you don't try and get the config from stagecraft and if you pass a -l (collector_slug) that you do.